src/main/java/uk/ac/cam/cl/dtg/isaac/dos/IsaacSymbolicChemistryQuestion.java

```diff
@@ -31,6 +31,8 @@
 public class IsaacSymbolicChemistryQuestion extends IsaacSymbolicQuestion {
     @JsonProperty("isNuclear")
     private boolean isNuclear;
+    @JsonProperty("allowPermutations")

```

Comment:
This isn't necessary! We need it for the line above since Jackson tries to do clever name things with "standard" getters and setters; `isABCDE` is assumed to be the getter for a JSON property called `ABCDE`, whereas we want the JSON property to actually be `isNuclear`.
I'd like to avoid these annotations generally so they don't proliferate unnecessarily like this . . .

Reply:
```suggestion
```

Reply:
(I wondered about "allow" being special, but reading between the lines of https://github.com/FasterXML/jackson-annotations/wiki/Jackson-Annotations#property-inclusion, I think that `get...`, `set...` and `is...` are the only "special" prefixes Jackson cares about).